galaxy/api/views/views.py

```diff
@@ -109,6 +117,71 @@ def filter_rating_queryset(qs):
 # -----------------------------------------------------------------------------
 
 
+class MonitorRootView(base_views.APIView):
+    """ Monitor resources """
+    permission_classes = (AllowAny,)
+
+    monitor_response = OrderedDict()
+
+    def get(self, request, format=None):
+        if not settings.GALAXY_METRICS_ENABLED:
+            self.monitor_response['influx'] = 'n/a'
+        else:
+            influxdb_client = influxdb.InfluxDBClient(
+                host=settings.INFLUX_DB_HOST,
+                port=settings.INFLUX_DB_PORT,
+                username=settings.INFLUX_DB_USERNAME,
+                password=settings.INFLUX_DB_PASSWORD
+            )
+
+            try:
+                influxdb_client.ping()
+                self.monitor_response['influx'] = 'ok'
+            except influxdb.client.InfluxDBClientError:
+                self.monitor_response['influx'] = 'error'
+
+            influxdb_client.close()
```

Comment:
This line may not be called in case of any unexpected exceptions above. Please use context managers (`with` statement), [contextlib.ExitStack](https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack) or `try .. finally`.
Because InfluxDBClient does not support context management protocol, ExitStack or `try .. finally` will be appropriate approach for guaranteed closing the connection.